config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -38,8 +38,31 @@ presubmits:
       testgrid-tab-name: release-cluster-up
       testgrid-alert-email: release-managers@kubernetes.io
       testgrid-num-columns-recent: '30'
+  - name: pull-release-test
+    alwasy_run: true
+    decorate: true
+    skip_report: true
```

Comment:
Do we want to just remove this and enforce now?

Reply:
Done